File: cf-1971b.cpp
#include <iostream>
#include <unordered_set>
using namespace std;

int main () {
    int t;
    cin >> t;

    while (t--) {
        string s;
        cin >> s;

        unordered_set <char> st;
        for (char c : s)
            st.insert(c);

        if (st.size() == 1) {
            cout << "NO\n";
            continue;
        }

        int len = s.length();
        for (int i = 1; i < len; i++) {
            char tmp = s[i];
            s[i] = s[i - 1];
            s[i - 1] = tmp;
        }

        cout << "YES\n" << s << '\n';
    }
}

File: cf-1971b.py
t = int(input())

while t:
    t -= 1
    s = list(input())
    
    if (len(set(s)) == 1):
        print("NO")
        continue
    
    for i in range(1, len(s)):
        tmp = s[i]
        s[i] = s[i - 1]
        s[i - 1] = tmp
        
    s = ''.join(s)
    print("YES\n" + s)